less/admin/UsersListPage.less

```diff
@@ -0,0 +1,127 @@
+.UserListPage {
+  // Pad bottom of page to make nav area look less squashed
+  padding-bottom: 24px;
+
+  &-grid {
+    width: 100%;
+    position: relative;
+    border-radius: @border-radius;
+
+    // Use CSS custom properties to define the number of columns in the grid
+    grid-template-columns: repeat(var(--columns), max-content);
+
+    // Ensure mobile scrollbar isn't on top of content
+    padding-bottom: 4px;
+
+    // Table refreshing overlay
+    &--loadingPage {
+      &::after {
+        content: "";
+        display: block;
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        background: rgba(128, 128, 128, 0.2);
+      }
+
+      .LoadingIndicator-container {
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+      }
+    }
+
+    &--loaded,
+    &--loadingPage {
+      display: grid;
+      overflow-x: auto;
+    }
+
+    &--header {
+      font-weight: bold;
+      border-bottom: 1px solid @muted-more-color;
+      padding: 8px 16px;
+      background: @control-bg;
+    }
+
+    &--rowItem {
+      padding: 4px 16px;
```

Comment:
The elements inside aren't well centered, let's add `display:flex; align-items:center;`